### PR TITLE
rOps changes to accommodate RFT specific messages

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/876SlD64/634-s36-user-account-creation-rops-changes-to-accommodate-rft-specific-messages

### CODE REVIEW

File: response_operations_ui/views/messages.py [hunk 1]
```diff
@@ -449,6 +451,76 @@ def close_conversation(thread_id):
...
+    limit = request.args.get('limit', default=10, type=int)
+    conversation_tab = request.args.get('conversation_tab', default='open')
+    ru_ref_filter = request.args.get('ru_ref_filter', default='')
+    business_id_filter = request.args.get('business_id_filter', default='')
+    category = "MISC"
+
+    form = SecureMessageRuFilterForm()
+
+    try:
+        if form.validate_on_submit():
```
Comment (warrenbailey) [inline response_operations_ui/views/messages.py:470]: Is it possible to refactor this method? 

It's pretty much repeated three times in the code and this function is almost an identical copy of the view_technical_inbox method but with a different category. 

Maybe we could craete one method which takes a category and that's called by two different view functions?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: response_operations_ui/templates/secure-message/misc-inbox.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        <span class="u-fs-r">(<a id="change-survey" href="{{ url_for('messages_bp.select_inbox') }}">Change inbox</a>)</span>
+      </h1>
+      {% if ru_ref_filter %}
+          <p id="clear_ru_ref" class="u-fs-s">Filtered by RU ref: {{ ru_ref_filter }}
+            <a href="{{ url_for('messages_bp.clear_filter', selected_survey=selected_survey, conversation_tab=conversation_tab) }}">Clear filter</a>
+          </p>
+      {% endif %}  
+    </div>
+    <div class="grid__col u-mt-s@s"> 
+      <form action="{{ url_for('messages_bp.view_technical_inbox', selected_survey=selected_survey, conversation_tab=conversation_tab, ru_ref_filter=ru_ref_filter, business_id_filter=business_id_filter, page=page, limit=limit) }}" method="post">
```
Comment (insacuri) [inline response_operations_ui/templates/secure-message/misc-inbox.html:44]: the url_for needs to be different here

**Review by insacuri — CHANGES_REQUESTED** [review]

File: response_operations_ui/templates/secure-message/misc-inbox.html [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        {% endfor %}
+      {% endcall %}
+    {% endif %}
+  {% endwith %}
+  <div class="grid grid--flex grid--between">
+    <div class="grid__col"> 
+      {% if breadcrumbs %}
+        {% include 'layouts/configs/breadcrumbs.html' %}
+      {% endif %}
+      <h1 class="u-fs-l u-mb-xs" name="page-messages-title">FDI Messages
```
[the remark this answers is not in the document]
Comment (creativeappdevamit) [reply response_operations_ui/templates/secure-message/misc-inbox.html]: yes you are right, don't know why I put FDI :-)

File: response_operations_ui/templates/secure-message/misc-inbox.html (hunk 2 above)
Comment (creativeappdevamit) [reply response_operations_ui/templates/secure-message/misc-inbox.html]: yes.. missed it totally. Copy paste error. fixed it

**Review by warrenbailey — APPROVED** [review]

**Review by insacuri — APPROVED** [review]

**Pull request merged**